master/reference/calicoctl/resources/bgppeer.md

```diff
@@ -30,31 +30,31 @@ spec:
 
 | Field       | Description                 | Accepted Values   | Schema |
 |-------------|-----------------------------|-------------------|--------|
-| scope    | Determines the Calico nodes to which this peer applies. | global, node | string |
-| node     | Must be specified if scope is node, and must be omitted when scope is global. | The hostname of the node to which this peer applies. | string |
-| peerIP   | The IP address of this peer. | Valid IPv4 or IPv6 address.  | string |
+| name     | Unique name to describe this resource instance. Must be specified.| Alphanumeric string with optional `.`, `_`, `-`, or `/` | string |
 
 #### Spec
 
 | Field       | Description                 | Accepted Values   | Schema | Default    |
 |-------------|-----------------------------|-------------------|--------|------------|
+| node     | If specified, the scope is node level, otherwise the scope is global. | The hostname of the node to which this peer applies. | string | |
+| peerIP   | The IP address of this peer. | Valid IPv4 or IPv6 address.  | string | |
 | asNumber | The AS Number of this peer. | A valid AS Number, may be specified in dotted notation. | integer/string |
 
 ### Peer Scopes
 
-BGP Peers can exist at either a global scope or a node scope.  A peer's scope 
+BGP Peers can exist at either a global scope or a node scope. A peer's scope 
 determines which Calico nodes will attempt to establish a BGP session with that peer.
 
 #### Global Peer
 
-If this is a `global` scoped BGP peer, all nodes in the cluster will attempt to 
-establish a BGP connection with it.
+If this is a global scoped BGP peer, i.e. when `node` field is not specified, all nodes in
+the cluster will attempt to establish a BGP connection with it.
```

Comment:
I think it is good that you are trying to clarify this, but the sentence is a little unwieldy now.... You should probably try to make it into two sentences. Perhaps something like: "To assign a BGP peer a global scope, omit the `node` field. All nodes in the cluster to attempt to establish BGP connections with it."

Reply:
done!